evm/evm.go

```diff
@@ -247,6 +253,29 @@ func (e Evm) NewEnv(origin common.Address) *vm.EVM {
 	return vm.NewEVM(e.context, e.sdb, &e.chainConfig, e.vmConfig)
 }
 
+func constantinopleChainConfig() params.ChainConfig {
```

Comment:
There's a one line difference between `defaultChainConfig` and `constantinopleChainConfig`, add a new parameter to `defaultChainConfig` to indicate which version of the EVM should be used so it's immediately obvious what the difference between the two versions is.